### PR TITLE
Send Login Links from the Editor

### PR DESCRIPTION
**Pull request by roosnic1**
This PR allows editor users to send Login links for the website to users. It also introduces a new method in the `URLAdapter` that builds the Website Login Link URL.

### CODE REVIEW

**Review by camilleory — APPROVED** [review]

File: packages/editor/src/client/panel/userEditPanel.tsx [hunk 1]
```diff
@@ -207,6 +210,23 @@ export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
                 <Button appearance="primary" onClick={() => setIsResetUserPasswordOpen(true)}>
                   {t('userList.panels.resetPassword')}
                 </Button>
+                <Button
+                  appearance="primary"
+                  style={{marginLeft: '20px'}}
+                  disabled={!email || !active}
+                  onClick={async () => {
```
Comment (penk85) [inline packages/editor/src/client/panel/userEditPanel.tsx:217]: Should it maybe be disabled here before result?

**Review by penk85 — APPROVED** [review]

File: packages/editor/src/client/panel/userEditPanel.tsx [hunk 2]
```diff
@@ -207,6 +210,23 @@ export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
...
+                      Alert.success(
+                        t('userList.panels.sendWebsiteLoginSuccessMessage', {email}),
+                        2000
+                      )
+                    } catch (error) {
+                      Alert.error(t('userList.panel.sendWebsiteLoginFailureMessage', {error}), 0)
+                    }
+                  }}>
+                  {t('userList.panels.sendWebsiteLogin')}
+                </Button>
```
Comment (lakeside763) [inline packages/editor/src/client/panel/userEditPanel.tsx:229]: May be line 218 to 226 can be separated into a method e.g handleSendWebsiteLogin

**Review by lakeside763 — COMMENTED** [review]
This looks good

File: packages/editor/src/client/panel/userEditPanel.tsx (hunk 1 above)
Comment (roosnic1) [reply packages/editor/src/client/panel/userEditPanel.tsx:217]: @penk85 good idea but with such a small task maybe a bit over engineered.

**Pull request merged**